frontend/package.json

```diff
@@ -12,7 +12,8 @@
     "dev": "yarn clean && yarn plugin-stats && ts-node -O '{\"module\":\"commonjs\"}' ./node_modules/.bin/webpack --mode development --watch --progress",
     "build": "yarn clean && yarn plugin-stats && NODE_ENV=production ts-node -O '{\"module\":\"commonjs\"}' ./node_modules/.bin/webpack --mode=production",
     "coverage": "jest --coverage .",
-    "lint": "eslint --ext .js,.jsx,.ts,.tsx --color .",
+    "eslint": "eslint --ext .js,.jsx,.ts,.tsx --color",
+    "lint": "yarn run eslint .",
```

Comment:
Nit: this can be simply `yarn eslint .` - any non-standard Yarn command is automatically treated as `yarn run <cmd>`.

Reply:
removed run